[Feat/TK-75]: 지출 상세 내역 퍼블리싱

yaejinkong opened the pull request:
## 📋 연관된 이슈 번호

- close #14 

## 🛠 구현 사항

- 정산 요청이 왔을 경우 보여줄 지출 상세 내역 페이지 퍼블리싱
- 정산 취소하기 모달 퍼블리싱

## 📚 변경 사항

## 🏜 스크린샷

- 상세내역 보기 버튼 클릭 전
<img width="351" alt="image" src="https://github.com/user-attachments/assets/51414836-b07c-46db-817e-1f492960c4c1">

- 상세 내역 보기 버튼 클릭 후
<img width="353" alt="image" src="https://github.com/user-attachments/assets/2b08a7cc-cf7b-4c68-a841-b688f9dd9a3d">

- 정산 취소하기 모달
<img width="350" alt="image" src="https://github.com/user-attachments/assets/c7708019-f221-4cc7-bbc4-738a0e8b4287">

## 💬 To Reviewers

- 피드백 부탁드려요!

yaejinkong committed:
[feat/TK-75]: 지출 상세 내역 퍼블리싱

yaejinkong committed:
[refactor/TK-75]: 상세내역 보기 버튼 수정

- Details 태그에서 button으로 변경
- 버튼 위치 변경

yaejinkong committed:
[refactor/TK-75]: 동의하지 않은 그룹원이 상단으로 가도록 변경

- 동의하지 않은 그룹원들은 리스트의 아래쪽으로 배치

yaejinkong committed:
[refactor/TK-75]: 동의한 그룹원의 카드 css 수정

yaejinkong committed:
[refactor/TK-75]: 디렉토리 구조 변경

yaejinkong committed:
[feat/TK-75]: Modal 컴포넌트 변경사항 직접 merge

yaejinkong committed:
[feat/TK-75]: 정산 취소하기 모달 퍼블리싱

doyi0107 commented:
내요말고 보내요는 어떤가요?

SeungAh-Yoo99 commented:
내가 보내줘야한다면 이도이'님에게'는 어떨까요?

그리고 다른 분들 이름도 볼드되어 있으면 더 잘보일 거 같아요!!

Kong-E commented on apps/mobile/app/travel/[id]/settlement/page.module.css:
```diff
@@ -0,0 +1,197 @@
+.mainCard {
+  border-radius: 10px;
+  padding: 25px 23px;
+  border: 1px solid #e5e7eb;
```
<img width="904" alt="image" src="https://github.com/user-attachments/assets/4a1090f5-3144-47a1-9d3c-3523cb94bc04">
<img width="405" alt="image" src="https://github.com/user-attachments/assets/c9c93cc9-ce58-49f7-b6f6-97445563e119">


제가 global css에 저희 팔레트에 있는 색상들은 이런 식으로 변수명을 지정해둬서 그걸 사용하면 좋을 것 같아요!

`border: 1px solid var(--color-gray-200);`

yaejinkong replied:
넵 변경하겠습니다!

yaejinkong commented:
- "위 금액은 정산 후 개인이 최종적으로 송금하거나 수령할 금액입니다." 멘트 추가
- 정산 완료 대신 동의 완료로 우선 수정 (추후 도장으로 변경할 예정)

Kong-E commented on packages/ui/src/modal.tsx:
```diff
@@ -22,13 +29,14 @@ export const Modal = ({ isOpen, onClose, title, children }: ModalProps) => {
+        onClick={(e) => e.stopPropagation()}
       >
         <header className={styles.header}>
           <h2>{title}</h2>
```
타이틀 폰트 사이즈 18px이나 20px로 줄이는 게 좋을 것 같아요!

yaejinkong replied:
dev 머지하면서 수정된 거 같습니다!!

Kong-E commented on packages/ui/src/modal.module.css:
```diff
@@ -1,45 +1,37 @@
+  background: white;
+  border-radius: 8px;
+  padding: 20px;
+  width: 375px;
```
기기마다 뷰포트 너비가 다르기 때문에 width를 375px로 고정하는 것보다 100%로 설정하고 margin을 양쪽에 10px 내외로 주는 게 어떤가요??!

yaejinkong replied:
dev 머지하면서 수정된 거 같습니다!!

yaejinkong commented:
> 내요말고 보내요는 어떤가요?

넵 수정하겠습니다!

yaejinkong commented:
> 내가 보내줘야한다면 이도이'님에게'는 어떨까요?
> 
> 그리고 다른 분들 이름도 볼드되어 있으면 더 잘보일 거 같아요!!

이름 볼드 처리하겠습니다!

Kong-E commented on apps/mobile/app/travel/[id]/settlement/page.module.css:
```diff
@@ -0,0 +1,197 @@
+  transform: rotate(90deg);
+}
+
+.arrowOpen {
```
여기도 .arrowClosed와 동일하게 `margin-left: 3px;` 줍시당
(.arrow에 주면 될 것 같네요!)

yaejinkong replied:
수정했습니다!!

yaejinkong committed:
Merge remote-tracking branch 'origin/dev' into feat/TK-75-expense-detail

Kong-E commented on apps/mobile/app/travel/[id]/settlement/page.tsx:
```diff
@@ -0,0 +1,224 @@
+                  isOpen ? styles.arrowOpen : styles.arrowClosed
+                }`}
+              >
+                &gt;
```
1) 화살표 아이콘도 되도록 통일하면 좋을 것 같아서 피그마에 있는 화살표 아이콘 그대로 써야할 것 같아요!
2) 화살표 아이콘은 크게 의미 있는 이미지가 아니기 때문에, "상세내역 접기/보기" 태그 부분에 `::after` css 가상셀렉터 써서 화살표를 표시하면 더 좋은 코드일 것 같아용

yaejinkong replied:
아하 일단 피그마에 있는 화살표로 바꾸었습니다!

yaejinkong committed:
[refacor/TK-75]: 지출 내역 상세 페이지 퍼블리싱 리팩토링

- 여행원 이름 bold 처리
- 당사자 이름 옆에 (나) 추가
- 내요 -> 보내요로 변경
- 피그마의 화살표 아이콘으로 변경
- 정산완료 -> 동의완료로 변경
- 설명 멘트 추가
- global.css의 색상 사용

yaejinkong committed:
Merge remote-tracking branch 'origin/dev' into feat/TK-75-expense-detail

yaejinkong committed:
[refactor/TK-122]: padding-bottomm, merge 누락 부분 수정

yaejinkong committed:
[refactor/TK-75]: 정산 취소 모달 css 수정

github-actions[bot] committed:
chore: apply Prettier formatting

yaejinkong committed:
[refactor/TK-75]: Modal 컴포넌트 css 수정

yaejinkong committed:
Merge remote-tracking branch 'origin/feat/TK-75-expense-detail' into feat/TK-75-expense-detail
